app/minecraft_api.rb

```diff
@@ -17,10 +17,15 @@ class MinecraftApi < Grape::API
 	  end
 
 	  desc "Returns a list of all types"
-	    get "/types" do
+	  get "/types" do
 	    MinecraftItems.all_types.to_json
 	  end
 
+	  desc "Returns a specific minecraft block 'type'"
```

Comment:
So this is actually going to be dynamic, its' going to look something like this

```
      desc 'Returns a specfic minecraft block type'
      get '/:material_type' do
        material_type = params[:material_type]
        MinecraftItems.minecraft_types[material_type]
      end
```

The `:` in the string means its supposed to be some kind of parameter if that makes sense.